modules/oe_content_event/src/Plugin/Validation/Constraint/EventFieldsRequired.php

```diff
@@ -26,4 +26,18 @@ class EventFieldsRequired extends Constraint {
    */
   public $message = '@name field is required.';
 
+  /**
+   * The error message shown when featured media is provided.
+   *
+   * @var string
+   */
+  public $mediaNotEmpty = 'The %legend and %body fields are required when %media is provided.';
```

Comment:
Why? All three fields were required if any of them was filled in and editors wanted to be able to provide only the body value so that's how I changed the validation. No validation if only body is filled in but still make all three required if media or legend are provided.